packages/api/src/module/write/email-confirmation/presentation/rest/email-confirmation.rest-controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Body, Controller, HttpCode, Inject, Post, UseGuards } from '@nestjs/common';
+import { CommandBus } from '@nestjs/cqrs';
+
+import { JwtUserId } from '@/crud/auth/jwt/jwt-user-id.decorator';
+import {
+  RequestEmailConfirmationApplicationCommand,
+  requestEmailConfirmationCommand,
+} from '@/module/commands/request-email-conformation';
+import { UserId } from '@/shared/domain.types';
+import { JwtAuthGuard } from '@/shared/guards/jwt-auth.guard';
+import { ApplicationCommandFactory } from '@/write/shared/application/application-command.factory';
+import { ID_GENERATOR, IdGenerator } from '@/write/shared/application/id-generator';
+
+@Controller('email-confirmation')
+export class EmailConfirmationRestController {
+  constructor(
+    private readonly commandBus: CommandBus,
+    private readonly commandFactory: ApplicationCommandFactory,
+    @Inject(ID_GENERATOR)
+    private readonly idGenerator: IdGenerator,
+  ) {}
+
+  @UseGuards(JwtAuthGuard)
+  @Post('')
+  @HttpCode(204)
+  async emailConfirmation(@JwtUserId() userId: UserId, @Body() body: { confirmationFor: string }): Promise<void> {
```

Comment:
It'd be nice to introduce class for RequestBody, because i think in the future we'd like to add some validation for requests. It's possible with https://docs.nestjs.com/techniques/validation